sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -137,7 +140,7 @@ private bool TryGetMethodOutputBinding(MethodDeclarationSyntax method, SemanticM
 
                 if (outputBindingAttribute != null)
                 {
-                    if (!TryCreateBindingDict(outputBindingAttribute, Constants.FunctionMetadataBindingProps.ReturnBindingName, bindingLocation, out IDictionary<string, object>? bindingDict))
+                    if (!TryCreateBindingDict(outputBindingAttribute, Constants.FunctionMetadataBindingProps.ReturnBindingName, bindingLocation, supportsDeferredBinding: false, out IDictionary<string, object>? bindingDict))
```

Comment:
Can you make supportsDeferredBinding false by default in the `TryCreateBindingDict` method